### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MarcelCoding**
https://discord.com/channels/454739001841025024/454750853967118336/856966453219557426 + followings ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ Nothing easier than that! Three steps to start:
 
 Cryptic supports many clients to play.
 The most popular is the client for your browser: [click here](https://play.cryptic-game.net/)
-But you can also play in an virtual terminal: [cryptic-term](https://github.com/cryptic-game/cryptic-term)
+But you can also play in an virtual terminal: [PyCrypCli](https://github.com/Defelo/PyCrypCli)
```
Comment (Tristan-H11) [inline README.md]:
```suggestion
But you can also play in a virtual terminal: [PyCrypCli](https://github.com/Defelo/PyCrypCli)
```

File: README.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ Nothing easier than that! Three steps to start:
...
 
 More clients are everywhere in the internet, so it's your choice.
```
Comment (Tristan-H11) [inline README.md]:
```suggestion
More clients are everywhere on the internet, so it's your choice.
```

**Review by Tristan-H11 — CHANGES_REQUESTED** [review]
Please correct these two typos. @MarcelCoding

**Comment by Tristan-H11** [comment]
@Defelo @DevMimas  Please approve if ok

**Review by Defelo — DISMISSED** [review]

**Comment by DevMimas** [comment]
Please delete the faq@cryptic-game.net is not in use

**Review by ClientCrash — CHANGES_REQUESTED** [review]
Please add a small text on the bottom that these rule (specifically the no-bots rule) applies only on the official backend and not self hosted ones.

**Comment by cadamini** [comment]
> Please delete the [faq@cryptic-game.net](mailto:faq@cryptic-game.net) is not in use

The commit removing the address will also close https://github.com/cryptic-game/cryptic/issues/50.

**Review by Tristan-H11 — APPROVED** [review]

**Review by DevMimas — APPROVED** [review]

**Review by Defelo — APPROVED** [review]

**Pull request merged**